Workiva/over_react_codemod
# RM-59778 Release over_react_codemod 1.4.2 (Hotfix for CPLAT-7950)

## Motivation
The React 16 codemods were choking in repos where there were `.g.dart` files generated using the Dart 1 SDK.

## Changes
Ignore `.g.dart` files when running the React 16 codemods.

#### Release Notes
Ignore `.g.dart` files when running the React 16 codemods

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: @greglittlefield-wf @joebingham-wk @kealjones-wk @sydneyjodon-wk 

### QA Checklist
        - Pull this down locally, and activate it using the `--source` flag.
        - Run `pub global run over_react_codemod:react16_upgrade --fail-on-changes` within `graph_ui` - and verify that it does not fail.

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:


[contributing-review-types]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#manual-testing-criteria
